### PR TITLE
How CART is substantial in speech emotion recognition

### PR DESCRIPTION
**Pull request by slynn22**
Revised version of the order

## Our **Peer Reviewer and Content Moderator teams do NOT provide any revisions services.** All revisions and edits should be completed on your own forked repo (as to not take up room in the queue) in order for our team to review them all in a timely manner. 

**ANY ARTICLE SUBMITTED WITH GLARING ERRORS WILL BE IMMEDIATELY CLOSED.**

>As a rule of thumb - please be sure to only submit articles (pull requests) that are fully polished and ready to be published. Be sure to go through our resources documents for extra and 3rd party (vetted) resources to help improve overall technical writing.

> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

`This closes #issue-number`

> Finally, delete the article checklist notes in blockquotes and submit your PR. We look forward to reviewing your article.

### CODE REVIEW

**Comment by slynn22** [comment]
@collins-a, this is the new PR of the order which you asked me to revise (#4887)

File: content/articles/how_cart_algorithms_are_substantial_in_speech_emotion_recognition/index.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+- \*\*CART also incorporates data tests and cross-validation\*\*. This is vital in ensuring that the goodness of fit is accurate. Thus, the decision tree created is authentic in representing the emotions represented in the speech.
+
+- \*\*CART is also efficient in using different variables in varying levels of the decision tree\*\*. This ensures that it is easy to unwrap complex interdependencies between variable sets. Thus, various emotional ties are identified in different levels of the decision tree.
+
+- \*\*CART in speech emotion recognition can be conjoined with other algorithms\*\*. The ability to use CART with other algorithms enhances speech emotion recognition accuracy. For example, CART can be utilized with [SVM](/engineering-education/supervised-learning-algorithms/) for outlier detection, classification, and regression.
```
Comment (collins-a) [inline content/articles/how_cart_algorithms_are_substantial_in_speech_emotion_recognition/index.md:100]: To bolden your sentences, replace all the `\*\*` in this section with `**`.

**Review by collins-a — CHANGES_REQUESTED** [review]
Hey @slynn22, thank you for revising the article. However, you still need to do the following:

1. You still need to add a **hero image**, an **author bio**, and **headshot/avatar image** as with the [PR](https://github.com/section-engineering-education/engineering-education/pull/4887) I closed and make sure they are correctly formatted.
2. Add the title to your new PR.
3. Edit the formatting on the section I've highlighted on the use of `**` as opposed to `*\*\`

And please expedite the changes for speedy publishing. Ping me when done.

File: content/articles/how_cart_algorithms_are_substantial_in_speech_emotion_recognition/index.md (hunk 1 above)
Comment (slynn22) [reply content/articles/how_cart_algorithms_are_substantial_in_speech_emotion_recognition/index.md:100]: Alright

**Comment by slynn22** [comment]
@collins-a revised as requested

**Comment by slynn22** [comment]
@collins-a, do I have to make more changes, if so which ones since it is still labeled "changes requested" and yet to be labeled approved?

**Comment by collins-a** [comment]
hey @slynn22, your article is ready for the final review. It's now upon the final reviewer to approve then get your article published

**Review by collins-a — APPROVED** [review]

**Comment by slynn22** [comment]
Thank you

**Comment by hectorkambow** [comment]
jumping on this one

**Comment by slynn22** [comment]
@hectorkambow , what do you mean?

**Comment by hectorkambow** [comment]
I will be performing a final review to see if this pull request can be merged (published) 👍

**Comment by slynn22** [comment]
@hectorkambow alright. Thank you.. i hope it will be published. Looking forward to a positive response from you

**Comment by hectorkambow** [comment]
@slynn22  

Please be sure to review any and all suggestions/changes made by your peer reviewer and/or myself during a pre and final reviews. 

All revisions and edits should be completed on your own forked repo (as to not take up room in the queue) in order for our team to review them all in a timely manner. ANY ARTICLE SUBMITTED WITH GLARING ERRORS WILL BE IMMEDIATELY CLOSED.

 
cc @collins-a  Please do the same 👍 as there were various issues throughout the pull request - please be sure that we are only adding the "ready for final review" label to pull request that are error/issue free & fully polished. 
For example:
The repo name should not include underscores - a few sentences were duplicates. 

One main thing to NOTE - lets please be sure to ping the @section-engineering-education/content-moderator-team moderator team when a pull request is ready for final and more than 48 hours have passed. This pull request had been "ready" for 17 days? It is our responsibilty to ping the team to make sure pull request are not left un-attended. 👍  Thank you !

**Comment by slynn22** [comment]
@hectorkambow, thank you for the response, should I do anything or just wait for the publishing?

**Comment by slynn22** [comment]
@hectorkambow, i noted you have already published. Thank you so much

**Comment by ahmadmardeni1** [comment]
When we merge a PR, it will be published automatically. @slynn22

**Comment by hectorkambow** [comment]
> @hectorkambow, thank you for the response, should I do anything or just wait for the publishing?

Our comments are for future reference 👍

**Comment by collins-a** [comment]
Noted @hectorkambow. Thank you

**Pull request merged**